src/modules/payment-app-configuration/input-schemas.ts

```diff
@@ -3,7 +3,7 @@ import { paymentAppFormConfigEntrySchema } from "./config-entry";
 
 export const mappingUpdate = z.object({
   channelId: z.string().min(1),
-  configurationId: z.string().min(1),
+  configurationId: z.string().min(1).nullable(),
```

Comment:
Why should this field be nullable? I think if you remove mapping the entry should be gone altogether